test/testlib/OpenSSL/Test.pm

```diff
@@ -279,52 +293,80 @@ An example:
 
 =cut
 
-sub app {
+sub cmd {
     my $cmd = shift;
     my %opts = @_;
     return sub { my $num = shift;
-		 return __build_cmd($num, \&__apps_file, $cmd, %opts); }
```

Comment:
__build_cmd is declared at the top of the file but never defined. It can probably go.